emcc.py

```diff
@@ -1284,8 +1284,13 @@ def check(input_file):
         # wasm backend output can benefit from the binaryen optimizer (in asm2wasm,
         # we run the optimizer during asm2wasm itself). use it, if not overridden
         if 'BINARYEN_PASSES' not in settings_key_changes:
+          parts = []
```

Comment:
nit: instead of just "parts" how about "passes" or "pass_args"

Reply:
Done.